src/components/VersionedCodeEditor/utils/utils.ts

```diff
@@ -0,0 +1,10 @@
+import { OCP_VERSION_4_18 } from './constants';
+
+const parseOcpVersion = (version?: string): number[] =>
+  version?.split('.').map((versionSection) => parseInt(versionSection, 10)) ?? [];
+
+export const isVersionGte = (actual?: string, target = OCP_VERSION_4_18): boolean => {
```

Comment:
I'm wondering if we want to default to 4.18 here. Maybe we don't need a default here at all and always require the usage of this to provide a target? 

I'm wondering what happens when OCP updates again and maybe 4.18 also leads to a breaking change.

Reply:
agreed removed default 4.18

Let's hope they won't introduce more of these breaking changes, I'm pretty confident that if we talk with console team, they might work a solution on their on their end for this, but it would take time